canonical-websites/www.ubuntu.com
# 639 openstack managed page

## Done

Update `openstack/managed`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/openstack/managed>
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#)


## Issue / Card

Fixes [#639](https://github.com/ubuntudesign/web-squad/issues/639)
